source/conf.py

```diff
@@ -54,7 +54,7 @@
 
 # General information about the project.
 project = 'QIIME 2 Developer Documentation'
-copyright = '2018, QIIME 2 Development Team'
+copyright = '2019, QIIME 2 Development Team'
```

Comment:
Since this project was started in 2018, how about we keep that info in the updated copyright? So, something like:

```
2018-2019
```